src/termwidget.cpp

```diff
@@ -138,7 +138,7 @@ void TermWidgetImpl::customContextMenuCall(const QPoint & pos)
     QMenu menu;
     QMap<QString, QAction*> actions = findParent<MainWindow>(this)->leaseActions();
 
-    QList<QAction*> extraActions = filterActions(pos, &menu);
+    QList<QAction*> extraActions = filterActions(pos); //, &menu);
```

Comment:
Error: no matching function for call to ‘TermWidgetImpl::filterActions(const QPoint&)’

Reply:
I got the opposite error. When I try to compile with &menu, it says only ‘TermWidgetImpl::filterActions(const QPoint&)’ exists. I'm not sure. But without those 2 changes the code won't compile on my machine. Also, sorry about the NULL vs nullptr. I am more comfortable with C styling over C++.

Reply:
> But without those 2 changes the code won't compile on my machine.

Do you have the latest git `qtermwidget`?

> sorry about the NULL vs nullptr

No need to be sorry. It was `0` before ;)

Reply:
Well I cloned the repo a few hours ago so I think I am up to date

Reply:
> Where is setMargin() defined?

https://github.com/lxqt/qtermwidget/blob/6fb3b06575bf160dd365e57db1ae54112c10be34/lib/qtermwidget.h#L226

Reply:
Ohhhh qtermwidget is a different package!!! ...yeah I didn't download that

Reply:
Reasons to update Arch

Reply:
For PRs, it's always necessary to have the latest git versions of the program *and all of its immediate dependencies*.

Reply:
This may be a while then

Reply:
Is there anything else which I should explicitly build?

Reply:
No.

Reply:
I'm having some issues with the prefixes and compiling. What do I do?
This is what I did:
git clone https://github.com/lxqt/qtermwidget.git
cd qtermwidget
mkdir build
cd build
cmake `pwd`/..
make
make install DESTDIR=~/